tests/suites/1_stateful/01_load/01_0000_streaming_load.sh

```diff
@@ -13,6 +13,16 @@ if [ ! -f /tmp/ontime.csv ]; then
 fi
 
 
+# do the Data integrity check
+sha256sum -c $CURDIR/ontime.sha1sum.txt
```

Comment:
As @sundy-li commented, we can use:

```
echo "429c47cdd49b7d75eec9b9244c8b1288edd132506203e37d2d1e70e1ac1eccc7 ontime.csv" | sha256sum --check
```

To check sha256 directly in shell without extra files.

Reply:
fix it